mentoring/templates/html/mentoring_grade.html

```diff
@@ -6,11 +6,38 @@
   <% }} %>
   <h2>You scored <%= score %>% on this assessment.</h2>
   <hr/>
-
+  <% var q, last_question; %>
   <span class="assessment-checkmark icon-2x checkmark-correct icon-ok fa fa-check"></span>
-  <p>You answered <%= correct_answer %> questions correctly.</p>
+  <div class="results-section"><p>You answered <%= correct_answer %> questions correctly<% if (enable_extended && correct.length) {{ %>:<% }} else {{ %>.<% }} %></p>
+    <% if (enable_extended) {{ var questions = correct; %>
+        <ul class="review-list correct-list">
+           <% for (var question in questions) {{ q = questions[question]; last_question = question == questions.length - 1; %>
```

Comment:
@Kelketek could this loop be abstracted into custom tag? If not, it still might make sense to create a function to fomat this kind of string and than pass it as a context variable